components/Navigation.js

```diff
@@ -92,65 +92,58 @@ export default function Navigation(props) {
-    >
+      <List component="nav" aria-labelledby="nested-list-subheader" className={classes.root}>
         {navNodes.map((node) => {
           // console.log(node.label)
-          var isSubMenItem = (node.label === 'Gallery')
-          var subMListItem = (node.label === 'Travel' | node.label === 'Videography')
+          var isSubMenItem = node.label === 'Gallery';
+          var subMListItem = node.label === 'Travel' || node.label === 'Videography';
           if (isSubMenItem) {
             return (
               <ListItem button key={node.label} onClick={handleClick}>
-                <ListItemText primary={node.label}/>
+                <ListItemText primary={node.label} />
                 {open ? <ExpandLess /> : <ExpandMore />}
-                  <Collapse in={open} timeout="auto" unmountOnExit>
-                    <List component="div" disablePadding>
-                      <Link href={'/travel'}>
-                        <ListItem button className={classes.nested}>
-                          <ListItemText primary="Travel" />
-                        </ListItem>
-                      </Link>
-                      <Link href={'/videography'}>
-                        <ListItem button className={classes.nested}>
-                          <ListItemText primary="Videography" />
-                        </ListItem>
-                      </Link>
-                    </List>
+                <Collapse in={open} timeout="auto" unmountOnExit>
+                  <List component="div" disablePadding>
+                    <Link href={'/travel'}>
+                      <ListItem button className={classes.nested}>
+                        <ListItemText primary="Travel" />
+                      </ListItem>
+                    </Link>
+                    <Link href={'/videography'}>
+                      <ListItem button className={classes.nested}>
+                        <ListItemText primary="Videography" />
+                      </ListItem>
+                    </Link>
+                  </List>
                 </Collapse>
               </ListItem>
-            );} else if(!isSubMenItem & !subMListItem) {
-              return (
-                <ListItem button key={node.label}>
+            );
+          } else if (!isSubMenItem & !subMListItem) {
+            return (
+              <ListItem button key={node.label}>
                 <Link href={node.path}>
                   <ListItemText primary={node.label}></ListItemText>
                 </Link>
-                </ListItem>
-              );
-            }
+              </ListItem>
+            );
+          }
         })}
-        </List>
+      </List>
+      <div>
+        <Link href="/cart">Cart</Link>
+      </div>
     </div>
   );
 
   // console.log(theme.breakpoints);
 
-  const container =
-    window !== undefined ? () => window().document.body : undefined;
+  const container = window !== undefined ? () => window().document.body : undefined;
```

Comment:
Should this be window().document or just window.document? Just curious, I haven't seen the former before

Reply:
Will change this, might have just been a VSCode suggestion oopsie.